### PR TITLE
ISSUE-398: fail if there are red lines in the Status Report admin page

### PR DESCRIPTION
**Pull request by beto-aveiga**
Adds a check that fails if there are errors in the Status Report page.

### CODE REVIEW

**Comment by beto-aveiga** [comment]
I tested it using other projects, and it worked.

**Review by mrdavidburns — APPROVED** [review]

File: scaffold/Taskfile.yml [hunk 1]
```diff
@@ -42,6 +42,11 @@ tasks:
     cmds:
       # @see https://architecture.lullabot.com/adr/20211026-one-time-login-first-drupal-user-account/
       - ./vendor/bin/drush user:unblock --uid=1
+      - |
+          [ $(drush core:requirements --severity=2 --format=tsv | wc -l) -lt 1 ] \
+          && echo "🟢 Status Report page checked" \
+          || false || echo "🔴 Status Report page check failed"
```
Comment (deviantintegral) [inline scaffold/Taskfile.yml:48]: What's this `|| false` for?

Comment (beto-aveiga) [reply scaffold/Taskfile.yml:48]: You are right 🤔 
I guess this wasn't needed...
Let me try without it.

**Review by deviantintegral — APPROVED** [review]

**Comment by deviantintegral** [comment]
This is probably worth calling out specifically in the release notes since it could cause working builds to fail.

**Comment by YesCT** [comment]
It's listed, but not called out in https://github.com/Lullabot/drainpipe/releases/tag/v3.12.1 cc @mrdavidburns

**Pull request merged**